pkg/server/engine.go

```diff
@@ -29,6 +30,9 @@ import (
 	"github.com/NVIDIA/topograph/pkg/utils"
 )
 
+var providerName string
```

Comment:
please avoid static variables. provider and engine might be passed in the payload.

Reply:
Fixed in last commit.